src/standup/standup.service.ts

```diff
@@ -62,6 +67,18 @@ export class StandupService {
             HttpStatus.NOT_FOUND,
           );
         }
+        if (
+          !this.validationService.validateAuthor(
+            data.author,
+            authorObject.uid,
+            authorObject.platform,
+          )
+        ) {
+          throw new HttpException(
+            "deletion failed: author doesn't match",
```

Comment:
I think we were are single quotes in the project? For example https://github.com/EddieHubCommunity/api/pull/112/files#diff-1cb9cb1e7db3765afcf2fe8e30efab22b562c878d991ad2b966dfe07447cf29bR95

Reply:
Single quotes wont work at this point because of the `doesn't` wheres the single quote

Reply:
We could escape it, if that helps with the consistency

```suggestion
            'deletion failed: author doesn\'t match',
```

Reply:
Just tested it. The current prettier-config we use turns it to the current implementation when escaping it.

Reply:
Ah ok 👍 thanks for testing